postgres/helm/postgres/templates/secrets.yaml

```diff
@@ -1,11 +1,13 @@
+{{- if .Values.configSecret.enabled }}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: postgres-environment
+  name: plural-postgres-s3
```

Comment:
we should probably keep this secret the same name, as moving it very likely will cause all the dbs we've deployed to restart.  The configmap change might do that as well, though.  This also means making it optional is probably a no-go.

Reply:
we should test it on a non-prod cluster to ensure this doesn't happen at least on aws so we don't bring down app.plural.sh for a bit with it